cmd/thanos/sidecar.go

```diff
@@ -117,6 +122,12 @@ func runSidecar(
 		uploads = false
 	}
 
+	readinessProber := prober.NewProber(comp, logger, prometheus.WrapRegistererWithPrefix("thanos_", reg))
```

Comment:
it is actually both health and ready probe, right?  So can we remove all `readiness` mentions here?

Reply:
Ok, I wasn't sure if name `prober` is understandable on it's own but you are right it's both even liveness. Will rename.

Reply:
it collided with the package name as a just `prober` so changed it to `statusProber` if that is ok with you?